housekeeper/store/models.py

```diff
@@ -4,16 +4,15 @@
 from pathlib import Path
 
 from sqlalchemy import Column, ForeignKey, Table, UniqueConstraint, orm, types
-from sqlalchemy.orm import declarative_base
-from sqlalchemy.orm import backref
+from sqlalchemy.orm import backref, declarative_base
 
 Model = declarative_base()
 
 file_tag_link = Table(
     "file_tag_link",
     Model.metadata,
-    Column("file_id", types.Integer, ForeignKey("file.id"), nullable=False),
-    Column("tag_id", types.Integer, ForeignKey("tag.id"), nullable=False),
+    Column("file_id", types.Integer, ForeignKey("file.id", ondelete="CASCADE"), nullable=False),
+    Column("tag_id", types.Integer, ForeignKey("tag.id", ondelete="CASCADE"), nullable=False),
```

Comment:
Just for my peace of mind: this only works one way right? It does not remove file entries or tag entries if a file_tag_link entry is removed?

Reply:
It does not remove the file, but it removes the tag from the list of tags in the file (`file.tags`). Maybe I will do a test on stage to show this

Reply:
I think it only involves the entries in this table (the file tag association table). So if a file is deleted, all entries in this table with its id is deleted and vice versa

Reply:
I did a test on stage that supports this, see below

Reply:
Looks great!